resources/scripts/pytest_otel/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Has the inclusion of this license been cleared with the legal folks?

Reply:
we are releasing everything on Apache License

https://github.com/jenkinsci/opentelemetry-plugin/blob/master/LICENSE
https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/maven-extension/src/main/java/io/opentelemetry/maven/package-info.java
https://github.com/kuisathaverat/pipeline-filebeat-logs/blob/main/LICENSE
https://github.com/elastic/apm-pipeline-library/blob/master/LICENSE